Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_datetimeperiod.py

```diff
@@ -862,7 +1085,7 @@ def parse_specific_time_of_day(self, source: str, reference: datetime) -> DateTi
 
         before_str = source[0:match[-1].start()].strip()
         after_str = source[match[-1].end():].strip()
-        er = next(iter(self.config.date_extractor.extract(
+        expected_result = next(iter(self.config.date_extractor.extract(
```

Comment:
It would be better to rename 'er' as 'extracted_result'.

Reply:
ACK. We will be renaming all 'er' variables to 'extracted_result', instead of 'expected_result'.